app/messaging/publish-event-request.js

```diff
@@ -0,0 +1,10 @@
+const { MessageSender } = require('ffc-messaging')
+const createMessage = require('./create-message')
+
+const publishEventRequest = async (eventMessage, config) => {
+  const eventSender = new MessageSender(config)
+  const message = createMessage(eventMessage, 'uk.gov.pay.event.publisher')
```

Comment:
Is it possible to use the event type from the `type` in `action`?  Otherwise all event types will be the same